src/hangman/Main.java

```diff
@@ -1,9 +1,252 @@
+			    "boggle",
+			    "bookworm",
+			    "boxcar",
+			    "boxful",
+			    "buffalo",
+			    "buffoon",
+			    "buxom",
+			    "buzzard",
+			    "buzzing",
+			    "buzzwords",
+			    "caliph",
+			    "cobweb",
+			    "croquet",
+			    "crypt",
+			    "curacao",
+			    "cycle",
+			    "daiquiri",
+			    "disavow",
+			    "dizzying",
+			    "duplex",
+			    "dwarves",
+			    "embezzle",
+			    "equip",
+			    "espionage",
+			    "exodus",
+			    "faking",
+			    "fishhook",
+			    "fixable",
+			    "fjord",
+			    "flapjack",
+			    "flopping",
+			    "fluffiness",
+			    "flyby",
+			    "foxglove",
+			    "frazzled",
+			    "frizzled",
+			    "fuchsia",
+			    "funny",
+			    "gabby",
+			    "galaxy",
+			    "galvanize",
+			    "glowworm",
+			    "glyph",
+			    "gnarly",
+			    "gnostic",
+			    "gossip",
+			    "grogginess",
+			    "haiku",
+			    "haphazard",
+			    "hyphen",
+			    "iatrogenic",
+			    "icebox",
+			    "injury",
+			    "ivory",
+			    "ivy",
+			    "jackpot",
+			    "jaundice",
+			    "jawbreaker",
+			    "jaywalk",
+			    "jelly",
+			    "jigsaw",
+			    "jinx",
+			    "jiujitsu",
+			    "jockey",
+			    "jogging",
+			    "joking",
+			    "jovial",
+			    "joyful",
+			    "juicy",
+			    "jukebox",
+			    "jumbo",
+			    "kayak",
+			    "kazoo",
+			    "keyhole",
+			    "khaki",
+			    "kilobyte",
+			    "kiosk",
+			    "lengths",
+			    "lucky",
+			    "luxury",
+			    "lymph",
+			    "marquis",
+			    "matrix",
+			    "megahertz",
+			    "microwave",
+			    "mnemonic",
+			    "mystify",
+			    "nowadays",
+			    "onyx",
+			    "oxidize",
+			    "oxygen",
+			    "pajama",
+			    "peekaboo",
+			    "phlegm",
+			    "pixel",
+			    "pizazz",
+			    "pneumonia",
+			    "polka",
+			    "psyche",
+			    "puppy",
+			    "puzzling",
+			    "quartz",
+			    "queue",
+			    "quiz",
+			    "quizzes",
+			    "quorum",
+			    "rhubarb",
+			    "rhythm",
+			    "rickshaw",
+			    "schnapps",
+			    "snazzy",
+			    "sphinx",
+			    "spritz",
+			    "squawk",
+			    "staff",
+			    "strength",
+			    "stretch",
+			    "stronghold",
+			    "stymied",
+			    "subway",
+			    "swivel",
+			    "thriftless",
+			    "topaz",
+			    "transcript",
+			    "transgress",
+			    "transplant",
+			    "twelfth",
+			    "twelfths",
+			    "unknown",
+			    "unworthy",
+			    "unzip",
+			    "uptown",
+			    "vaporize",
+			    "vodka",
+			    "voodoo",
+			    "vortex",
+			    "walkway",
+			    "waltz",
+			    "wave",
+			    "wavy",
+			    "waxy",
+			    "wellspring",
+			    "whiskey",
+			    "whizzing",
+			    "witchcraft",
+			    "wizard",
+			    "wyvern",
+			    "xylophone",
+			    "yachtsman",
+			    "youthful",
+			    "yummy",
+			    "zephyr",
+			    "zigzag",
+			    "zipper",
+			    "zodiac",
+			    "zombie"};
+		
+		String word = wordsList[(int) (Math.random() * wordsList.length)];
```

Comment:
This could be contained in a class. 
eg 
```java
public class WordList { 
 private String[] words = {"example", "another"} 
  
  public String getRandomWord() {
   // logic for getting word goes here 
  } 
 
}

Reply:
This way the logic isn't tied to the Main class